generate-certs/main.tf

```diff
@@ -47,17 +46,17 @@ resource "aws_iam_role_policy" "s3_gen_certs" {
         "s3:List*"
       ],
       "Resource": [
-        "arn:aws:s3:::${element(split("/", var.s3_root_path), 0)}"
```

Comment:
root path was also used to try and anticipate cases where the certs wasn't going to be mounted at the base of an s3 bucket.